### DISCUSSION TITLE
Governance UI - first impressions

### DISCUSSION THREAD

**Discussion by wirednkod** [jeluard/swipealot#453] [Ideas]
Very nice work everyone and thank you;

I have my concerns thought for the front page of the page; There seems to be a great amount of space that exists in the frontpage, and it is fine for someone who visits for the 1st time but other than that it seems unecessery (see screenshot below):
![image](https://user-images.githubusercontent.com/5408605/235674386-7e4f3849-2182-4ea2-8ede-a1faa9d04a72.png)

I think that this should be a "welcome" popup that can be dismissed, and rest of the screen should be a dashboard usable for as it is meant to be;

The "Volunteer Delegates" should not be a carousel but maybe a grid 3x3 so more information exist inside the page and not overflowing
 in 1 row;

**Comment by jeluard** [comment]
Once you have some active delegations, this section is hidden. It could also be hidden if you come multiple times to the app, but have no delegates yet.

**Reply by wirednkod** [reply]
IMO - this could be an one-time-see and a pop-up on demand.
Overall the UI looks close to the Polkadot UIs (e.g. like Polkadot Staking Dashboard - using Polka fonts, colors etc) but still needs a lot of UX and beautfications

**Reply by erler** [reply]
I can see something like a one-time modal with a few steps that walk you through the process

1Password walkthrough as a reference:
![1pw_onboarding1](https://user-images.githubusercontent.com/7072141/236779369-58cc44d1-7301-4df6-a869-045faae850b9.jpg)

Github Desktop reference:
<img width="536" alt="Github_Desktop-Onboarding" src="https://user-images.githubusercontent.com/7072141/236779553-a421017b-6afa-4da0-81ae-1d50c28719c8.png">

**Reply by dnlklmn** [reply]
we could add an onboarding modal to start with eventually, but i'd like to understand why you think its so important:
- both 1pwd and github provide vastly more features than the delegation app does
- our features are check/uncheck tracks, select a delegate, and sign a transaction (advanced feature is to add an address)
- so far we haven't heard anyone complain about not knowing what to do (except not having a clue about governance itself), that would justify having to give more help on how to use the app
- regarding governance related quesitons: our working theory is you don't just end up on the delegation app, you arrive from either a blog post, or the open gov landing page, so we assume you have some basic knowledge before arriving to the app. we will continuously monitor where product marketing and the landing page go, and track usage to see drop off points. at this point we don't have significant amount of data
- for people that come to this page a 2nd time after not delegating the 1st time (in which case they wouldn't see the intro again), we can assume they have _some_ doubts, therefore need _some_ further help. this first section will evolve and contain links to open gov related articles/tutorials, as they get ready

that said, we can make it disappear on a second visit (although we don't know why you didn't delegate the first time), or we can move it to a modal (although we don't know what people need help with just yet), but I personally see other features more pressing, eg. being able to link to a list of all delegates, or helping delegates provide usable information about themselves in a simple manner

**Reply by wirednkod** [reply]
I think the solution is far simpler; All that is needed is a "Dismiss" button that the user can click and "hide" the information;
When hidden a small tip can appear in case someone wants to revisit the information, which in case clicked, will reshow the information;

There is a range of cases, on why someone did not act on the 1st visit (e.g. just checking the site, mistakenly landed on the page etc);

**Reply by erler** [reply]
All I said is "I can see something like this", not that it is a priority or that you should do it. Like Nikos said, dismiss or hide function would be an easy option. Also – this is just a guess – I can imagine a higher chance of a user interacting when they see already a few actionable UI elements (even though disabled, as the wallet is not connected)

> so far we haven't heard anyone complain about not knowing what to do (except not having a clue about governance itself), that would justify having to give more help on how to use the app

Not sure this true. There was some confusion with what the different tracks actually mean

**Reply by dnlklmn** [reply]
making disabled buttons actionable is planned

re:what tracks actually mean, i don't know if explaining what each track means in detail is the app's job. there was some hierarchy issues i can remember (difficult to differentiate between descriptions of tracks and referenda on tracks), those we'll address soon
is there something else we're missing?

**Reply by wirednkod** [reply]
IMO Governance site (and every "product" site) should have help pages on, flows, explanations, links for explaining processes, etc etc; ATM there are no explanations on: what is governance, how delegation works, what is the benefits, what can I do after delegate, what tracks mean etc etc...

**Comment by dnlklmn** [comment]
Thanks for the quick feedback @wirednkod ,

We'll keep reworking the upper part as we learn more about our users and as product marketing refines their narrative around open gov in general. 

At this point we assume everyone (that doesn't delegate using polkadotjs already) need basic info to feel comfortable delegating.

Re:needs ux, what do you think could be improved?
Where in the process did you feel confused or lost?

If you can spare 45 min it would be amazing if you could do a user test with us!

**Reply by wirednkod** [reply]
There are 2 parts of such an app should have:
a) a "Learn/Read more" section that can contain, guides, articles, videos etc about Governance
b) a "Use the dashboard for actions" section

At the moment the Page has both parts in same page, putting all users under the same group; 
Does not matter if the user is an advanced one or a new one; both will end up in the same page;
To me the top part and the scrolling carousel was annoying;
I can provide more feedback when https://github.com/paritytech/governance-ui/issues/450 is fixed

**Comment by gilescope** [comment]
Looks good for selecting a delegate but can't see how to set the conviction? Would be great if that was clearer. widescreen / firefox / linux - clearly a common setup ;-)

**Reply by jeluard** [reply]
Conviction can't be set for now. That's something we are considering for next iterations.

**Comment by gilescope** [comment]
Given that it cannot be set I would only be delegating 0.1x of my vote.
That makes me think I should not delegate if I can’t set that as they
wouldn’t haven enough oomph to make a difference.
